Robot2019/src/main/java/frc/robot/OI.java

```diff
@@ -7,35 +7,30 @@
 
 package frc.robot;
 
+import java.util.ArrayList;
+import java.util.Collection;
+
 import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
-  public Joystick leftJoy = new Joystick(1);
-  public Joystick rightJoy = new Joystick(2);
-
-  // Button button = new JoystickButton(stick, buttonNumber);
-
-  // There are a few additional built in buttons you can use. Additionally,
-  // by subclassing Button you can create custom triggers and bind those to
-  // commands the same as any other Button.
-
-  //// TRIGGERING COMMANDS WITH BUTTONS
-  // Once you have a button, it's trivial to bind it to a button in one of
-  // three ways:
-
-  // Start the command when the button is pressed and let it run the command
-  // until it is finished as determined by it's isFinished method.
-  // button.whenPressed(new ExampleCommand());
-
-  // Run the command while the button is being held down and interrupt it once
-  // the button is released.
-  // button.whileHeld(new ExampleCommand());
-
-  // Start the command when the button is released and let it run the command
-  // until it is finished as determined by it's isFinished method.
-  // button.whenReleased(new ExampleCommand());
+  public Joystick leftJoy = new Joystick(getPort("LeftJoystick"));
+  public Joystick rightJoy = new Joystick(getPort("RightJoystick"));
+  public ArrayList<Joystick> joysticks = new ArrayList<Joystick>() {{ add(leftJoy); add(rightJoy); }};
```

Comment:
Is there a reason to have these in an ArrayList? Sometimes they may need to be passed separately and also having an ArrayList runs the risk of mixing up the indices.